go/staking/api/api.go

```diff
@@ -520,6 +520,95 @@ func (p *ConsensusParameters) SanityCheck() error {
 	return nil
 }
 
+// SanityCheck does basic sanity checking on the genesis state.
+func (g *Genesis) SanityCheck() error {
```

Comment:
Should this enforce the new lower limit added in #2366?

Reply:
It looks like that is already sufficiently enforced when adding the escrow, so not sure if it would be beneficial to duplicate the check here.  Also, if the parameter changes, the genesis document might become unloadable on a dump/restore if it were enforced on this level.

Reply:
Ok.